### ISSUE TITLE
Bindgen removes necessary constants

### ISSUE THREAD

**Issue by nptr** [norepimorphism/libui-ng-sys#4]
### Observed behaviour:
libui-ng-sys does not expose all enums/constants. Among many, `uiAlignFill` is missing.

### Expected behaviour:
All necessary enums/constants are available.

### Likely problem:

1. libui-ng defines anonymous enums via their `_UI_ENUM()` macro. See [uiAlign](https://github.com/libui-ng/libui-ng/blob/master/ui.h#L2951) for example. 
3. `bindgen` normally generates a type for each enum. Anonymous enums get something along the line `_bindgen_ty_16`.
```rust
pub type uiAlign = ::std::os::raw::c_uint;
pub const uiAlignFill: ::std::os::raw::c_uint = 0;
pub const uiAlignStart: ::std::os::raw::c_uint = 1;
pub const uiAlignCenter: ::std::os::raw::c_uint = 2;
pub const uiAlignEnd: ::std::os::raw::c_uint = 3;
pub type _bindgen_ty_16 = ::std::os::raw::c_uint;
```
5. libui-ng-sys blacklists those bindgen generated types and (implicitly?) the accompanying constants in [build.rs:473](https://github.com/norepimorphism/libui-ng-sys/blob/main/build.rs#L473). Leaving just this in the resulting bindings.rs file:

```rust
pub type uiAlign = ::std::os::raw::c_uint;
```

### Possible solution
The quick fix would be to not remove `_bindgen` items in the build script.
I assume there is a cleaner solution, but I am not familiar with `bindgen`.

**Comment by norepimorphism** [comment]
Thanks for the issue!

I blacklisted the `_bindgen_ty_*` items because they seemed to add no value, but I did not realize that doing so implicitly blacklisted the accompanying constants as well. I suppose it makes sense from `bindgen`'s point-of-view, though.

I can confirm that removing this line fixes the issue, as you mentioned:
https://github.com/norepimorphism/libui-ng-sys/blob/f05bd7f28aa2256373d2dbf083182c12ef3511e4/build.rs#L503

I will look at the `bindgen` documentation to see if it provides a way to approach this issue. Otherwise, I will commit your suggestion as it is more important for constants like `uiAlignFill` to be available than for a few extra `_bindgen_ty_*` items to be visible.

**Comment by norepimorphism** [comment]
OK, I accidentally included your suggested fix in an unrelated commit before researching alternative solutions, but I'm going to keep it for now because it seems to resolve the original problem. As such, I will mark this issue as **closed**. If my latest commit does not fix the problem for you, please re-open the issue.

Because this issue is now closed, I have created a new issue (#5) to continue discussion on alternative solutions.

**Comment by nptr** [comment]
Thanks a lot!
